contracts/command-batcher/src/state.rs

```diff
@@ -1 +1,27 @@
+use axelar_wasm_std::Threshold;
+use connection_router::types::ChainName;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, Uint256, Uint64};
+use cw_storage_plus::{Item, Map};
 
+use crate::types::{BatchID, CommandBatch, ProofID};
+
+#[cw_serde]
+pub struct Config {
+    pub admin: Addr,
+    pub gateway: Addr,
+    pub multisig: Addr,
+    pub service_registry: Addr,
+    pub destination_chain_id: Uint256,
+    pub signing_threshold: Threshold,
+    pub service_name: String,
+    pub chain_name: ChainName,
```

Comment:
I don't think we can use `ChainName` here, since that's a type that's meant to be validated when converting from string.

Reply:
in that case I think it's ok `ChainName` here since this is the state struct, instead I changed it to String in the Instance msg and validated during instantiation before storing it in the config